src/http/controllers/user/nurseUser.js

```diff
@@ -53,7 +53,33 @@ const getSingleNurseUser = async (req, res) => {
   }
 };
 
+/**
+ * @description Controller for "get all nurse user" API operation
+ * @param {object} req Express request object
+ * @param {object} res Express response object
+ */
+const getAllNurseUser = async (req, res) => {
+  const { log, user, userType } = res.locals;
+  const hospitalId = userType === 'hospital_admin' ? user._id : user.hospitalId;
+  const wardId = userType === 'ward_user' ? user._id : user.wardId;
```

Comment:
Please make use of the user type constants from the db module as opposed to hard coding the user type string.